locale/texts.pot

```diff
@@ -434,7 +435,7 @@ msgid "Try again"
 msgstr ""
 
 #: src/screens/LoadWalletErrorScreen.js:21
-#: src/screens/PinScreen.js:265
+#: src/screens/PinScreen.js:268
```

Comment:
Why is this line being changed? You didn't change anything in `PinScreen`, am I missing something?

Reply:
Oh! I jut ran the translation script to get it updated.

Reply:
Ok, so `dev` was outdated

Reply:
I will run, but it may have happened during the many rebases.

Reply:
I checked your script in the dev:
1. extract the patch:
```sh
git format-patch -1 90607cb1f682eb4fe2d6ae8330cb7155e1254221
```
2. apply:
```sh
git apply 0001-refactor-check_pot-now-checks-for-missing-translatio.patch
```
3. run check:
```sh
make check_missing_translations
```
4. output:
```txt
✔ [ttag] translations extracted to /var/.../new.pot
check_pot: Checking missing translations on pot file...
```